### ISSUE TITLE
Feature request: Google map option in contact info

### ISSUE THREAD

**Issue by Norwayman** [e107inc/e107#1415]
From the discussion on http://www.e107help.org/2148/google-map-option-in-contact-info
I do not know if this is the right place to ask, but would someone who knows programming be willing to make a contact page plugin where one has the option to include a Google Map?
Either from coordinates, from address or embed google maps code, or use API, or special scripts etc?

Moc says it could be done rather easlily, but I do not know how to program. I just see things from the user side of it, and for me it seems that it could be useful for many users.

Thanks.

**Comment by nesjett** [comment]
Here its explained, very easy, https://developers.google.com/maps/tutorials/fundamentals/adding-a-google-map


Cheers,
- Nes

**Comment by Norwayman** [comment]
Yes, if you know programing this is probably easy, but for someone who does not it is a lot easier to have it done from the backend where you just fill in the information :)

**Comment by Norwayman** [comment]
Thank you Nestor, I will do that while waiting on the plugin :)

**Comment by nesjett** [comment]
I forgot that on v2 you have to paste it on a menu, but on the "code editor" mode, I mean when you are editing the menu, you will have to click on the button for "code" or "html", which is located on the text editing options (near the font size, font type, inser link, etc)

If youre lost Ill try to update a screenshot as soon as possible

**Comment by Jimmi08** [comment]
Just from curiosity. Why you just don't embed maps in contact info pref or menu? Why do you mess with script? Any reason for it?

![image](https://cloud.githubusercontent.com/assets/5429548/13698244/193ab7a8-e773-11e5-845e-f9a1943b5aec.png)

**Comment by nesjett** [comment]
It´s another aproach, there is no special reason, I just copied the first code I found on google for doing this.

Anyway, there is no much difference from pasting an iframe or pastin an script, at the end the iframe will be calling the same sources you are calling from the script.

**Comment by Jimmi08** [comment]
Thank you for explanation. I was asking because I know it's not so easy put script in tinymce (via button source code), not tested latest version.

**Comment by CaMer0n** [comment]
I have an old v2 plugin for this, it probably needs a little cleaning up.

**Comment by Jimmi08** [comment]
@CaMer0n Could you publish it please?

**Comment by Norwayman** [comment]
Thanks for all the answers and recommendations. I followed Jimmy08's way of doing it with an Iframe. Worked like a charm. After 10 years of using E107 I didn't know I could just paste this code directly into the contact info prefs :) For me the issue could be closed, as it works for me, but I still think it would be helpful for people not knowing e107 that one could just paste in the required info into options in a plugin type solution.
I also inserted the Iframe in a menu, but for those not knowing that one has to Insert Video to get to where one embeds, it is kind of tricky. Maybe a new tab saying EMBED would be more intuitive. Just my thoughts. Thanks! :)
